packages/server/postgres/migrations/1677272969994_meetingTemplatesMove.ts

```diff
@@ -0,0 +1,107 @@
+import {FirstParam} from 'parabol-client/types/generics'
```

Comment:
now that it's written I hope we can more or less copy/paste this for every table that we migrate